src/agent/cobertura/Cargo.toml

```diff
@@ -6,4 +6,8 @@ license = "MIT"
 
 [dependencies]
 anyhow = "1.0"
-quick-xml = "0.30"
+quick-xml = { git = "https://github.com/tevoinea/quick-xml/", branch = "tevoinea/AsyncElementWriter", features = [
```

Comment:
This is only temporary until my PR gets reviewed and merged. https://github.com/tafia/quick-xml/pull/635